src/Types/Output/ObjectType.hack

```diff
@@ -42,4 +47,16 @@ abstract class ObjectType extends NamedOutputType {
 
         return $is_valid ? new GraphQL\ValidFieldResult($ret, $errors) : new GraphQL\InvalidFieldResult($errors);
     }
+
+    public function getFields(bool $include_deprecated = false): ?vec<GraphQL\Introspection\IntrospectableField> {
+        // TODO
```

Comment:
My feeling is that we should use `getFieldDefinition` to resolve fields for both introspection and normal querying. Is the plan to make `FieldDefinition` implement `IntrospectableField`?

Reply:
Yup, this ended up working well